mixins/localize/localize-mixin.js

```diff
@@ -1,70 +1,29 @@
 import '@formatjs/intl-pluralrules/dist-es6/polyfill-locales.js';
```

Comment:
Can this polyfill import be removed from this file?

Reply:
Maybe, but when we add Maori we might need to add it back if it has it or add our own, which I'd prefer to do in `Intl`.

Reply:
Even though it's also included in `localize.js`?

Reply:
Oh! Derp. Yeah, I'll remove this.